### PR TITLE
Add support for auBankAccount element

### PR DESCRIPTION
**Pull request by wshallum-stripe**
### Summary & motivation

Add support for the (beta) auBankAccount component (depends on stripe/stripe-js#6).

### API review

A new `AuBankAccountElement` is added.

### Testing & documentation

Manually tested. Documentation is planned to be part of the beta docs.

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -173,6 +173,30 @@ export type PaymentRequestButtonElementComponent = React.FC<
...
 
+export interface AuBankAccountElementProps extends ElementProps {
+  /**
+   * An object containing [Element configuration options](https://stripe.com/docs/js/elements_object/create_element?type=paymentRequestButton).
+   */
+  options?: stripeJs.StripeAuBankAccountElementOptions;
+
+  /**
+   * Triggered when data exposed by this Element is changed (e.g., when there is an error).
+   * For more information, refer to the [Stripe.js reference](https://stripe.com/docs/js/element/events/on_change?type=ibanElement).
```
Comment (kabir-stripe) [inline src/types/index.ts]: `type=ibanElement` should probably be `type=auBankAccountElement`

File: src/types/index.ts [hunk 2]
```diff
@@ -230,5 +254,14 @@ declare module '@stripe/stripe-js' {
     getElement(
       component: PaymentRequestButtonElementComponent
     ): stripeJs.StripePaymentRequestButtonElement | null;
+
+    /**
+     * Returns the underlying [element instance](https://stripe.com/docs/js/elements_object/create_element?type=card) for the `IdealBankElement` component in the current [Elements](https://stripe.com/docs/stripe-js/react#elements-provider) provider tree.
```
Comment (kabir-stripe) [inline src/types/index.ts]: `type=card` and `IdealBankElement`

Comment (wshallum-stripe) [reply src/types/index.ts]: fixed!

File: src/types/index.ts (hunk 1 above)
Comment (wshallum-stripe) [reply src/types/index.ts]: fixed!

File: src/types/index.ts [hunk 3]
```diff
@@ -173,6 +173,30 @@ export type PaymentRequestButtonElementComponent = React.FC<
...
+
+
+  /**
+   * Triggered when the Element is fully rendered and can accept imperative `element.focus()` calls.
+   * Called with a reference to the underlying [Element instance](https://stripe.com/docs/js/element).
+   */
+  onReady?: (element: stripeJs.StripeAuBankAccountElement) => any;
+}
+
+export type AuBankAccountElementComponent = React.FC<
```
Comment (hofman-stripe) [inline src/types/index.ts]: If merging this after #40, please update to `ReactType`

Comment (hofman-stripe) [reply src/types/index.ts]: I merged #40 so this should be changed to:

```suggestion
export type AuBankAccountElementComponent = FunctionComponent<
```

Comment (wshallum-stripe) [reply src/types/index.ts]: done

**Comment by hofman-stripe** [comment]
Closing in favor of #41

**Pull request closed without merging**